scania-digital-design-system/tegel
# feat(dropdown): added support for normalized text when searching

**Describe pull-request**  
Requested in the support channel to have the dropdown search for accent answers for international users. opt-out able.

**Solving issue**  
Fixes: [CDEP-3001](https://tegel.atlassian.net/browse/CDEP-3001)

**How to test**  
1. Run storybook
2. Go do dropdown
3. Enable filter
4. Start searching for iles or aland
5. îles Åland should show up in the dropdown list
6. Turn off Normalized Text in storybook
7. Do the same search again and notice that its harder to find the dropdown item in the list.

**Suggested test steps**
- [x] Added/Updated unit tests that needs to be checked.

**Additional context**  
This is my first PR working with stencil and the this big storybook that you guys have so I'm sure that I've missed something or named something super wrong.


[CDEP-3001]: https://tegel.atlassian.net/browse/CDEP-3001?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
